org.gnome.Evince.json

```diff
@@ -164,16 +227,24 @@
             "buildsystem": "meson",
             "config-opts": [
                 "-Dnautilus=false",
-                "-Dthumbnailer=false"
+                "-Dgtk_doc=false",
+                "-Dintrospection=false",
+                "-Dthumbnailer=false",
+                "-Duser_doc=false"
```

Comment:
This flags should be a separate PR, or at least an independent commit. That said, the stable version does not need to match the development one. In the development version we skip the documentation because it runs in the CI after every commit, and we want to make it fast. This is for stable distribution of Evince in flathub.

Reply:
Then I'll revert this one